src/App.js

```diff
@@ -44,11 +54,13 @@ class App extends React.Component {
           completed: false,
         }
       ];
-      
+      const updatedTodos = this.state.todos.concat(newTodo);
       this.setState({
-        todos: this.state.todos.concat(newTodo),
+        todos: updatedTodos,
         newTodo: ''
       })
+      localStorage.removeItem('todos');
+      localStorage.setItem('todos', JSON.stringify(updatedTodos))
```

Comment:
- Nice job on the assingment Chinedu. You are properly handling state and all the events in React. You split Your app into smaller components to make them modular. Your todo app is fully functional and it is possible to add new tasks, mark them as complete and remove them.
- Your design looks really good, nice job with this one 👍
- Not even that but you made all the stretch task functional
  - local storage implemented
  - search functionality working
  - todo list deployed with netlify
- Your git workflow is really fluid and detailed, keep up the good job with this.